Fix ESMF_SL_LIBOPTS build
Updated easybuild/easyconfigs/e/ESMF/ESMF-6.1.1_libopts.patch
--- build/common.mk.orig	2025-11-20 10:17:24.432909000 +0100
+++ build/common.mk	2025-11-20 10:17:34.569510000 +0100
@@ -1211,7 +1211,7 @@
 ESMF_SL_LIBS_TO_MAKE  = libesmf
 endif
 ESMF_SL_LIBLINKER     = $(ESMF_CXXCOMPILER)
-ESMF_SL_LIBOPTS      +=
+ESMF_SL_LIBOPTS      += $(ESMF_CXXCOMPILEOPTS)
 ESMF_SL_LIBLIBS      +=
 
 # - Shared objects
